### DISCUSSION TITLE
Disable the drag & drop event

### DISCUSSION THREAD

**Discussion by olivier-nolf** [mokhosh/filament-kanban#36] [General]
Hi,

Is there by any change an option to disable the drag & drop event based on authorization?
So some users can change the status of the items and others can only view (and create a new one)

Kr,
Olivier

**Comment by mokhosh** [comment]
Hey,

This is not currently part of the api, but you can still achieve this on different levels.
For example if you want to disable drag and drop altogether, you can customise `kanban-board.blade` view, and render the scripts only under the condition that the user is authorised.
